json_spec/output/root.json

```diff
@@ -0,0 +1,75 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://github.com/opencomputeproject/ocp-diag-core/output",
+    "title": "OutputArtifact",
+    "description": "TODO: link to spec md link",
+    "properties": {
+        "sequenceNumber": {
+            "description": "TODO: link to spec md link",
+            "type": "integer",
+            "minimum": 0,
+            "maximum": 2147483647
```

Comment:
We didn't bother specifying integer overflow for the other integers defined in this spec. Should be consistent and either remove this or add it for the other indices.

Reply:
done, please close after check